utils/travis/check_utf8.sh

```diff
@@ -1,13 +1,41 @@
 #!/bin/bash
-# Install isutf8 program (from package "moreutils" at least in linux mint)
-# in order to use this script
+# Install isutf8 program (from package "moreutils" at least in linux mint) in order to use this script.
 #
-# This script assumes that the current working directory is the root of the
-# wesnoth repository.
-set -e
+# This script assumes that the current working directory is the root of the wesnoth repository.
+EXIT_CODE=0
+
 find src/ -type f -print0 | xargs -0 isutf8 --
+if [ $? != 0 ]; then
```

Comment:
The `if` should be written as: `|| exit_code=1` instead. (Use upper case variable names only for environment variables.)